hack/generator/pkg/codegen/pipeline/status_from_swagger_test.go

```diff
@@ -57,11 +59,77 @@ func Test_ShouldSkipDir_GivenPath_HasExpectedResult(t *testing.T) {
 		c := c
 		t.Run(c.name, func(t *testing.T) {
 			t.Parallel()
-			g := NewGomegaWithT(t)
+			g := gomega.NewWithT(t)
 
 			skipped := shouldSkipDir(c.path)
 
-			g.Expect(skipped).To(Equal(c.shouldSkip))
+			g.Expect(skipped).To(gomega.Equal(c.shouldSkip))
```

Comment:
Didn't we standardise on using implicit+imports + `NewGomegaWithT()` to minimize stuttering through the exports?